internal/config/generator.go

```diff
@@ -93,6 +97,11 @@ type JiraCLIConfigGenerator struct {
 		mtls         struct {
 			caCert, clientCert, clientKey string
 		}
+		oauth struct {
+			accessToken  string
+			refreshToken string
+			cloudId      string
```

Comment:
You are using cloudID everywhere else


```suggestion
			cloudID      string
```